VideoAPI/VideoApi.IntegrationTests/Database/Commands/SaveMonitoringCommandTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using VideoApi.DAL;
+using VideoApi.DAL.Commands;
+using VideoApi.Domain;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.IntegrationTests.Database.Commands
+{
+    public class SaveMonitoringCommandTest : DatabaseTestsBase
+    {
+        private SaveMonitoringCommandHandler _handler;
+        private Guid _newConferenceId;
+
+        [SetUp]
+        public void Setup()
+        {
+            var context = new VideoApiDbContext(VideoBookingsDbContextOptions);
+            _handler = new SaveMonitoringCommandHandler(context);
+            _newConferenceId = Guid.Empty;
+        }
+
+        [Test]
+        public async Task should_save_monitoring()
+        {
+            var seededConference = await TestDataManager.SeedConference();
+            TestContext.WriteLine($"New seeded conference id: {seededConference.Id}");
+            _newConferenceId = seededConference.Id;
+            var participantId = seededConference.GetParticipants().First().Id;
+            
+            var command = new SaveMonitoringCommand(_newConferenceId, participantId, 0,0,0,0,0,0,0,0);
+            await _handler.Handle(command);
+
+            Monitoring savedMonitor;
+            await using (var db = new VideoApiDbContext(VideoBookingsDbContextOptions))
+            {
+                savedMonitor = await db.Monitorings.FirstOrDefaultAsync(x =>
+                    x.ConferenceId == _newConferenceId && x.ParticipantId == participantId);
+            }
+
+            savedMonitor.Should().NotBeNull();
+            savedMonitor.ConferenceId.Should().Be(_newConferenceId);
+            savedMonitor.ParticipantId.Should().Be(participantId);
+            savedMonitor.OutgoingAudioPercentageLost.Should().Be(0);
+            savedMonitor.OutgoingAudioPercentageLostRecent.Should().Be(0);
+            savedMonitor.IncomingAudioPercentageLost.Should().Be(0);
+            savedMonitor.IncomingAudioPercentageLostRecent.Should().Be(0);
+            savedMonitor.OutgoingVideoPercentageLost.Should().Be(0);
+            savedMonitor.OutgoingVideoPercentageLostRecent.Should().Be(0);
+            savedMonitor.IncomingVideoPercentageLost.Should().Be(0);
+            savedMonitor.IncomingVideoPercentageLostRecent.Should().Be(0);
```

Comment:
might be worth asserting the timestamp is not the default value

Reply:
ok cool